docs/content/en/docs/triggers/message-queue-trigger/message-queue-trigger-kind-keda.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Message Queue Trigger Kind Keda"
+date: 2020-07-20T14:30:01+05:30
+draft: true
+---
+
+# Brief Intro
+
+Message Queue Trigger component has following limitations:
+
+- For every new integration you want to enable - there was one pod running for enabling the integration
+- Autoscaling of the trigger handler was not available.
+- Lastly - one Fission installation could connect to and handle only one instance of a MQ. So for example you could only connect to one Kafka instance in a given Fission installation.
+
+When we decided to build out these features we came across [KEDA project](https://keda.sh) and it solved most of the limitations which Fission had in the event integration area. To understand the working of KEDA refer to the [KEDA documentation](https://keda.sh/docs/1.5/concepts/).
```

Comment:
We probably don't need this here as it is mentioned in blog

Reply:
Done